src/main/java/com/vinylteam/vinyl/util/impl/DetailedVinylParserImpl.java

```diff
@@ -16,7 +16,7 @@
 @RequiredArgsConstructor
 public class DetailedVinylParserImpl implements DetailedVinylParser {
 
-    private final ParserConfiguration config;
+    protected final ParserConfiguration config;
```

Comment:
Why protected? You will need access to this object from inheritor which is located in different package? If not, please you package-private(default) access modifier.